spec/lib/tasks/edition_updater_spec.rb

```diff
@@ -0,0 +1,67 @@
+RSpec.describe Tasks::EditionUpdater do
+  before do
+    stub_any_publishing_api_put_content
+    allow(EditDraftEditionService).to receive(:call).and_call_original
+    allow(FailsafeDraftPreviewService).to receive(:call).and_call_original
+  end
+
+  describe ".call" do
+    let(:change_history_id) { SecureRandom.uuid }
+
+    let(:change_history) do
+      [
+        {
+          "id" => change_history_id,
+          "public_timestamp" => "2020-03-01T00:00:00+00:00",
+          "note" => "Initial note",
+        },
+      ]
+    end
+
+    let(:edition) { create(:edition, locale: "en", change_history: change_history) }
```

Comment:
```suggestion
    let(:edition) { create(:edition, locale: "en") }
```

I don't think we do anything with this change history?

Reply:
Yes, it is useless - will remove